apps/election-manager/src/utils/saveAsPDF.test.ts

```diff
@@ -64,11 +64,11 @@ test('file path name is manipulated properly', async () => {
         'test-report-final-franklin-county-general-election-precinct-name.pdf',
     },
   ]
-  await testCases.forEach(async function ({ prefix, precinctName, expected }) {
+  for (const { prefix, precinctName, expected } of testCases) {
```

Comment:
@carolinemodic this was a fix for your tests where the old code wasn't properly `await`ing the calls to `saveAsPDF`.

Reply:
oh woops thanks!